.github/workflows/build.yml

```diff
@@ -44,4 +44,4 @@ jobs:
           EXTRA_TWISTER_FLAGS="--short-build-path -O/tmp/twister-out"
         fi
         west twister --test sample.basic.helloworld -p siwx917_rb4338a -v --inline-logs $EXTRA_TWISTER_FLAGS
-        west twister --test sample.net.wifi -p siwx917_rb4338a -v --inline-logs $EXTRA_TWISTER_FLAGS
+        west twister --test sample.net.wifi -p siwx917_rb4338a -v --inline-logs -K $EXTRA_TWISTER_FLAGS
```

Comment:
Just as a note for the future, twister supports multiple `--test` arguments, so we should likely just merge these into a single `west twister --test ... --test ...` command. I've postponed doing it since we have several PRs open which would conflict with such a change.

Another improvement would be to take advantage of GitHub workflow "matrix", so that GitHub would run in parallel the same workflow with different input parameters for each instance. The parameter could e.g. the a specific platform to build against, i.e. this becomes more relevant as we grow the list of platforms covered by our CI.

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow